cluster/events.go

```diff
@@ -0,0 +1,20 @@
+package cluster
+
+import "github.com/docker/docker/api/types/events"
+
+// Event is exported
+type Event struct {
+	events.Message
+	Engine *Engine `json:"-"`
+}
+
+// EventHandler is exported
+type EventHandler interface {
+	Handle(*Event) error
```

Comment:
Seems like you're right. I can't recall why it was this way in the past, because generally event related errors don't cause things to fail, but perhaps in the future we might want to log those errors. What do you think?